exercises/exercise-3.js

```diff
@@ -7,10 +7,36 @@
   Considerations: Values can be nested and datypes Number, String or Object.
 */
 
-function deepStringSum() {
-  // TODO: implement
+function deepStringSum(obj) {
+	let sum = 0;
+	for (key in obj) {
+		if(typeof obj[key] === 'object'){
+			sum += deepStringSum(obj[key]);
+		} else if (typeof obj[key] === 'string'){
+			sum += obj[key].length;
+		} 
+	}
+	return sum;
 }
 
+/* Traté de hacerlo con pura recursividad pero se me cicla
+function deepStringSum(obj, sum = 0) {
+	while(Object.entries(obj).length !== 0){
```

Comment:
When you're using recursion probably you won't need loops, probably this is the reason because you were having infinity loops